### DISCUSSION TITLE
[answered] Just a little suggestion about color of the arrows

### DISCUSSION THREAD

**Discussion by bencresty** [plausible/analytics#1034] [Anything Else]
Hello Guys,

Looking through analytic; just a little suggestion; Would it be possible to make the UP-arrows in the interface GREEN instead of RED? (on bounce rate, total page views etc. on top of the view) I think that would make it more instant readable and makes more sense if the down-arrows are red, like they are now, but the up-arrows will be green. Thanks for considering!

**Comment by Vigasaurus** [comment]
The arrows are colored by the context. For example, a higher bounce rate is a worse metric than a lower bounce rate, thus an increasing bounce rate is a worsening of stats. i.e. more bounces is worse than less bounces. 

A green arrow means the stat has improved, and a red means it has regressed - the direction of the arrow shows the actual direction of the change though.

**Reply by bencresty** [reply]
@Vigasaurus Ah, I see, you're completely right, than it makes much more sense indeed! Thanks for the quick response
